admin.py

```diff
@@ -0,0 +1,45 @@
+class Admin:
+    def __init__(self):
+        self.drinks = {}
+        self.dict_employees = {}
+
+    def add_drink(self, drinks: dict, title: str, price: int):
```

Comment:
If you use typing, use it everywhere. Label each attribute and the expected result of the function

Reply:
I don`t understand your comment

Reply:
I used typing in each function in the admin file